modules/veteran/app/models/veteran/flagged_veteran_representative_contact_data.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Veteran
+  class FlaggedVeteranRepresentativeContactData < ApplicationRecord
+    self.table_name = 'flagged_veteran_representative_contact_data'
+
+    attribute :flag_type, :flag_type
+    validates :ip_address, :representative_id, :flag_type, :flagged_value, presence: true
+    validates :flag_type, inclusion: { in: ActiveRecord::Type.lookup(:flag_type).valid_types, message: 'Invalid flag type: must be phone, email, address, or other' } # rubocop:disable Layout/LineLength,Rails/I18nLocaleTexts
```

Comment:
I think a cleaner implementation is defining `flag_type` as a string defined enum on this model

Reply:
TY. I got it working.